workflow.txt

```diff
@@ -1,10 +1,9 @@
+fork on github
 git clone
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
+git checkout master
 git add .
+git commit -m ""
 git pull upstream master
-git merge my-solution
-fork on github
-git checkout master
 git push origin my-solution
+git merge my-solution
```

Comment:
No need to merge here since you're going to push the code up and take care of merging branches through a pull request.